### DISCUSSION TITLE
Event on triggering ((field deactivation neutralizer)).

### DISCUSSION THREAD

**Discussion by IIMOHAXII** [EDCD/EDDI#2497] [EDDI Standalone Q&A] [answered]
Event on triggering ((field deactivation neutralizer)). that is, the Targoids, after knocking out the third heart, launch an impulse and a message appears on the screen - "Attention: an energy wave has been detected." If this message pops up (appears), is there information about it in the event log or not? And how to extract it?

**Comment by Tkael** [comment, marked as answer]
Unfortunately, I'm not aware of any data written to the journal for this.

**Reply by IIMOHAXII** [reply]
thanks
